ml4h/arguments.py

```diff
@@ -336,6 +336,9 @@ def parse_args():
     # TensorMap prefix for convenience
     parser.add_argument('--tensormap_prefix', default="ml4h.tensormap", type=str, help="Module prefix path for TensorMaps. Defaults to \"ml4h.tensormap\"")
 
+    # Special
+    parser.add_argument('--is_segmentation', default=False, help='Trigger segmentation-specific functionality.')
```

Comment:
I think we can remove this and use tensormap interpretation and shape check instead

Reply:
see below